its/projects/UTProjectProbing/UTProjectProbing.MsTstWithProjectPropertyFalse/calculator.cs

```diff
@@ -0,0 +1,4 @@
+﻿namespace UTProjectProbing.MsTstWithProjectPropertyFalse
```

Comment:
typo in project and namespace name
```suggestion
﻿namespace UTProjectProbing.MsTestWithProjectPropertyFalse
```

Reply:
This was indented, in order not to have `Test` in name.

Reply:
ah... ok... a comment would have helped :)